app/views/samples/show.html.erb

```diff
@@ -1,7 +1,7 @@
 <div id="PipelineSampleReads_page">
   <%= javascript_tag do %>
     react_component('PipelineSampleReads', {
-      admin: '<%= current_user.admin ? 1 : 0 %>',
+      admin: '<%= current_user.admin ? 1 : 0 %>' === '1',
```

Comment:
this can just be `admin: <%= current_user.admin == 1 %>` I think if you want the bool